tests/queries/0_stateless/02454_disable_mergetree_with_lightweight_delete_column.sql

```diff
@@ -0,0 +1,18 @@
+drop table if exists t_row_exists;
+
+create table t_row_exists(a int, _row_exists int) engine=MergeTree order by a; --{serverError 44}
+
+create table t_row_exists(a int, b int) engine=MergeTree order by a;
+alter table t_row_exists add column _row_exists int; --{serverError 44}
+alter table t_row_exists rename column b to _row_exists; --{serverError 44}
```

Comment:
alter table t_row_exists drop column b to _row_exists; ?

Reply:
I think it's okay to allow to drop this column, if user created a table with this column name.

Reply:
I tested drop column and rename column for _row_exists, an exception was reported as following: 
executeQuery: Code: 10. DB::Exception: Wrong column name. Cannot find column `_row_exists` to rename. (NOT_FOUND_COLUMN_IN_BLOCK)
executeQuery: Code: 10. DB::Exception: Wrong column name. Cannot find column `_row_exists` to drop. (NOT_FOUND_COLUMN_IN_BLOCK) (version 22.10.1.1) (from [::1]:37702) (in query: alter table t drop column _row_exists;),